src/Parsers/ASTKillQueryQuery.h

```diff
@@ -13,6 +13,11 @@ class ASTKillQueryQuery : public ASTQueryWithOutput, public ASTQueryWithOnCluste
     {
         Query,      /// KILL QUERY
         Mutation,   /// KILL MUTATION
+        // TODO(nv): THIS_KILL_TASK_TYPE naming is ugly.
```

Comment:
Why is it ugly? There are not so many things we can kill. I don't think that somebody actually moves parts `TO DISK` or `TO VOLUME` by hand.